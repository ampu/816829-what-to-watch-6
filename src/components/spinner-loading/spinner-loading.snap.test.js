import React from 'react';

import {render} from '@testing-library/react';

import SpinnerLoading from './spinner-loading';

describe(`SpinnerLoading component`, () => {
  it(`Should render according to the snapshot`, () => {
    const {container} = render(<SpinnerLoading/>);
    expect(container).toMatchSnapshot();
  });

  it(`Should render according to the snapshot when isInverse equals to true`, () => {
    const {container} = render(<SpinnerLoading isInverse/>);
    expect(container).toMatchSnapshot();
  });
});
