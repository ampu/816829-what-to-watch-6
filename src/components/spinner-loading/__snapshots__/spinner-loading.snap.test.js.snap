// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpinnerLoading component Should render according to the snapshot 1`] = `
<div>
  <div
    class="spinner-loading"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;

exports[`SpinnerLoading component Should render according to the snapshot when isInverse equals to true 1`] = `
<div>
  <div
    class="spinner-loading spinner-loading--inverse"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
